src/Orchard.Web/Modules/Orchard.Projections/Services/ProjectionManager.cs

```diff
@@ -120,9 +120,12 @@ public class ProjectionManager : IProjectionManagerExtension {
                 tokens.Add("Content", part.ContentItem);
             }
 
-            // aggregate the result for each group query
-            return GetContentQueries(queryRecord, Enumerable.Empty<SortCriterionRecord>(), tokens)
-                .Sum(contentQuery => contentQuery.Count());
+            var contentQueries = GetContentQueries(queryRecord, Enumerable.Empty<SortCriterionRecord>(), tokens);
+
+            if (queryRecord.FilterGroups.Count <= 1)
+                return contentQueries.Sum(contentQuery => contentQuery.Count());
+            else 
+                return contentQueries.SelectMany(contentQuery => contentQuery.ListIds()).Distinct().Count();
```

Comment:
I have a fetish for the ternary-operator, to be honest, but besides that, I think this would be a bit cleaner:

```suggestion
                return queryRecord.FilterGroups.Count > 1 ?
                    contentQueries.SelectMany(contentQuery => contentQuery.ListIds()).Distinct().Count() :
                    contentQueries.Sum(contentQuery => contentQuery.Count());
```

I added this comment with the suggestion feature for the better syntax highlighting, but you won't be able to correctly apply it directly, because this change corresponds to multiple lines in your code.

Reply:
@jimasp I added a suggestion for each (4) line involved, so you can commit them in a single batch.